<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KOnlineBankingStatusDecl</class>
 <widget class="QWidget" name="KOnlineBankingStatusDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>435</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="accountTab">
      <attribute name="title">
       <string>Account Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>STATUS:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout">
           <item>
            <widget class="KLed" name="m_ledOnlineStatus">
             <property name="state">
              <enum>KLed::Off</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="m_textOnlineStatus">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
               <horstretch>10</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Unavailable</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>41</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>BANK/BROKER:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="m_textBank">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&amp;lt;Not configured&amp;gt;</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>ACCOUNT:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="m_textOnlineAccount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&amp;lt;Not configured&amp;gt;</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="m_storePassword">
           <property name="text">
            <string>Store password</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KLineEdit" name="m_password">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
           <property name="passwordMode">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>143</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ofxTab">
      <attribute name="title">
       <string>OFX Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_1" columnstretch="0,2,2,0,1">
         <item row="1" column="1">
          <widget class="KComboBox" name="m_headerVersionCombo"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel1_2">
           <property name="text">
            <string>Identify as</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="KLineEdit" name="m_applicationEdit"/>
         </item>
         <item row="0" column="4">
          <spacer name="spacer4_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>150</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel1_3">
           <property name="text">
            <string>Header Version</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="KLineEdit" name="m_headerVersionEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="KComboBox" name="m_applicationCombo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="m_clientUidLabel">
           <property name="text">
            <string>Client UID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>User Agent</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QLineEdit" name="m_clientUidEdit">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>OFX &lt;CLIENTUID&gt; for institutions that require it</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="4">
          <widget class="QLineEdit" name="m_userAgentEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>195</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="importTab">
      <attribute name="title">
       <string>Import Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="buttonGroupBox2">
         <property name="title">
          <string>Start date of import</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item row="0" column="0" colspan="2">
              <widget class="QRadioButton" name="m_lastUpdateRB">
               <property name="text">
                <string>Last &amp;update</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup2</string>
               </attribute>
              </widget>
             </item>
             <item row="0" column="2" colspan="3">
              <widget class="QLabel" name="m_lastUpdateTXT">
               <property name="text">
                <string/>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QRadioButton" name="m_todayRB">
               <property name="text">
                <string>To&amp;day minus</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup2</string>
               </attribute>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="m_numdaysSpin">
               <property name="maximum">
                <number>180</number>
               </property>
              </widget>
             </item>
             <item row="1" column="3" colspan="2">
              <widget class="QLabel" name="textLabel2_2">
               <property name="text">
                <string comment="@action number of days">days</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="m_pickDateRB">
               <property name="text">
                <string>Pi&amp;ck date</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup2</string>
               </attribute>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="KMyMoneyDateInput" name="m_specificDate">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>214</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="buttonGroupBox1">
         <property name="title">
          <string>Import options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="2,0,0,0,0">
          <item row="0" column="1">
           <widget class="QComboBox" name="m_preferredPayee">
            <item>
             <property name="text">
              <string notr="true">PAYEEID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">NAME</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">MEMO</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Method to detect duplicate transactions during import</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="m_uniqueTransactionId">
            <item>
             <property name="text">
              <string>OFX FITID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KMyMoney Hash</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel3_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Payee's name is based on contents of the OFX tag</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Timezone offset</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="hl2" stretch="0,1">
            <item>
             <widget class="KComboBox" name="m_timestampOffsetSign">
              <item>
               <property name="text">
                <string>+</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>-</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="m_timestampOffset"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Invert sign of transaction amount</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="m_invertAmount">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check this if you need to invert the sign of imported transaction amounts, e.g. if you need to reverse charges and payments because your institution provides them inverted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fix sign of the investment transaction amount and quantity</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="m_fixBuySellSignage">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check this if the investment transactions after importing have incorrect Buy/Sell type assigned, which may be caused by your institution not applying the correct negative/positive sign to the share amount or quantity value.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyDateInput</class>
   <extends>QFrame</extends>
   <header>kmymoneydateinput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>m_storePassword</tabstop>
  <tabstop>m_password</tabstop>
  <tabstop>m_applicationCombo</tabstop>
  <tabstop>m_applicationEdit</tabstop>
  <tabstop>m_headerVersionCombo</tabstop>
  <tabstop>m_headerVersionEdit</tabstop>
  <tabstop>m_clientUidEdit</tabstop>
  <tabstop>m_userAgentEdit</tabstop>
  <tabstop>m_lastUpdateRB</tabstop>
  <tabstop>m_todayRB</tabstop>
  <tabstop>m_numdaysSpin</tabstop>
  <tabstop>m_pickDateRB</tabstop>
  <tabstop>m_preferredPayee</tabstop>
  <tabstop>m_uniqueTransactionId</tabstop>
  <tabstop>m_timestampOffsetSign</tabstop>
  <tabstop>m_timestampOffset</tabstop>
  <tabstop>m_invertAmount</tabstop>
  <tabstop>m_fixBuySellSignage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pickDateRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_specificDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_todayRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_numdaysSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_storePassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup2"/>
 </buttongroups>
</ui>
